src/views/accounts/RiskDetect/AccountCheckTaskList.vue

```diff
@@ -130,7 +129,7 @@ export default {
         hasExport: false,
         hasImport: false,
         createRoute: 'AccountCheckCreateUpdate',
-        canCreate: vm.$hasPerm('accounts.add_checkaccountautomation')
+        canCreate: vm.$hasPerm('accounts.add_checkaccountautomation') && !this.$store.getters.currentOrgIsRoot
       }
     }
   }
```

Comment:
Based on my current knowledge cut-off of 2021-09-01, it does not seem to have specific issues in terms of coding style or functional correctness. However, here's one potential improvement that could be considered for the `formatterArgs` key-value pair in your `actions`. It might look better structured with semicolon (;) at the end of every key value pair. Here is a suggestion:

```
export default {
    ...
    formatterArgs: {
      // This can update
    },
    ...
};
```